pkg/broker/instance_operations.go

```diff
@@ -174,6 +174,13 @@ func (b *Broker) createResources(ctx context.Context, client *mongodbatlas.Clien
 		}
 	}
 
+	for _, i := range dp.Integrations {
+		_, _, err := client.Integrations.Create(ctx, p.ID, i.Type, i)
```

Comment:
[q] am I right that we don't support update operations in AOSB?

Reply:
Also as we don't have tests for this - can you please put some output to the PR description proving that this works?

Reply:
Right, these integrations are created on project creation and never updated by design, just like DBUsers & IPAccessLists